app/javascript/packages/password-toggle/password-toggle-element.ts

```diff
@@ -29,6 +31,12 @@ class PasswordToggleElement extends HTMLElement {
   setInputType() {
     this.elements.input.type = this.elements.toggle.checked ? 'text' : 'password';
   }
+
+  showPasswordButtonClick() {
+    this.elements.toggle.addEventListener('click', () => {
+      trackEvent('Button clicked', { path: window.location.pathname });
```

Comment:
These events would come through to CloudWatch as "Frontend: Button clicked".

Can we contextualize the name to the Password Toggle component?

Also, you can avoid the "Frontend" prefix by adding the name to the event mapping here:

https://github.com/18F/identity-idp/blob/b55f0582918b849ca647cea471d436e4c373d4fe/app/controllers/frontend_log_controller.rb#L8-L33